.github/workflows/build-test-distribute.yaml

```diff
@@ -54,9 +96,8 @@ jobs:
       - name: Run container structure test
         if: ${{ !contains(github.event.pull_request.labels.*.name, 'ci/skip-container-structure-test') && !contains(github.event.pull_request.labels.*.name, 'ci/skip-test') }}
         run: |
-          make test/container-structure
+          ENABLED_GOARCHES=$RUNNER_ARCH make test/container-structure
```

Comment:
why is this test not under `test` job? or even a separate job just like in `circleci`

Reply:
Because only container images related steps will fail and other steps actual work across different arches.

Reply:
Charly suggested a way to support cross builds for those container related steps. So this is to be removed.